lua/gitlab/git.lua

```diff
@@ -93,39 +147,45 @@ M.contains_branch = function(current_branch)
   return run_system({ "git", "branch", "-r", "--contains", current_branch })
 end
 
----Returns true if `branch` is up-to-date on remote, false otherwise.
+---Returns true if `branch` is up-to-date on remote, otherwise false and warns user
 ---@param log_level integer
----@return boolean|nil
+---@return boolean
 M.current_branch_up_to_date_on_remote = function(log_level)
```

Comment:
To make it consistent with `check_mr_in_good_condition`:
```suggestion
M.check_current_branch_up_to_date_on_remote = function(log_level)
```